src/main/java/com/google/api/codegen/config/GapicProductConfig.java

```diff
@@ -371,7 +386,7 @@ public static GapicProductConfig createDummyInstance(
 
   /** Creates an GapicProductConfig with fixed content. Exposed for testing. */
   @VisibleForTesting
-  public static GapicProductConfig createDummyInstance(
+  private static GapicProductConfig createDummyInstance(
```

Comment:
If this can be private, should we remove the `@VisibleForTesting` annotation?

Reply:
done.